lib/cdo/graphics/certificate_image.rb

```diff
@@ -10,23 +10,28 @@ def create_certificate_image2(image_path, name, params={})
   name = name.to_s.force_8859_to_utf8.gsub(/@/, '\@').strip
   name = ' ' if name.empty?
 
+  # Load the certificate template
   background = Magick::Image.read(image_path).first
 
+  # The user's name will be put into an image with a transparent background.
+  # This uses 'pango', the OS's text layout engine, in order to dynamically
+  # select the correct font. This is important for handling non-latin
+  # languages.
+  pango_markup = "pango:<span foreground='#575757' font_weight='bold' font_family='Times'>#{name}</span>"
```

Comment:
do we need to do anything to ensure pango is available on our servers?

Reply:
Also curious at this.